Sclark/rubocop fixes add badge

sclark007 opened the pull request:
* fix gem ordering for rubocop errors
* add travis badge to readme.

sclark007 committed:
# This is a combination of 2 commits.
# This is the 1st commit message:

fix gem ordering

# This is the commit message #2:

fixing gem order, updating gemfile, add badge

sclark007 committed:
fix gem ordering

sclark007 committed:
add travis badge

sclark007 committed:
fixing gem order, updating gemfile, add badge

sclark007 committed:
update travis ruby versions

sclark007 committed:
update readme

sclark007 committed:
fix version format

jjasghar commented:
Seems reasonable, can you squash these commits down? 

With our new policy please use the [DCO signing process](https://discourse.chef.io/t/a-developer-certificate-of-origin-dco-is-now-required-with-code-contributions/9579).

For a [detailed description](http://stackoverflow.com/questions/13457203/how-to-add-the-signed-off-by-field-in-the-git-patch) or you can just do:

```
$ git commit -s
$ #Or
$ git commit --signoff
```

If you already have the commit, use git commit --amend or git rebase -i to edit your commit message, and add the above signoff line.

Or if you want a more perminant option the following is a way to set it forever:

```bash
$ git config commit.gpgsign true
```

jjasghar commented on .travis.yml:
```diff
@@ -2,9 +2,9 @@ language: ruby
-- 2.0.0
-- 2.1
-- 2.2
+  - 2.1.10
```
Please remove this from the `.travis.yml`

sclark007 committed:
Merge branch 'sclark/rubocop_fixes_add_badge' of github.com:sclark007/redhat-subscription-manager-cookbook into sclark/rubocop_fixes_add_badge

sclark007 committed:
Merge branch 'sclark/rubocop_fixes_add_badge' of github.com:sclark007/redhat-subscription-manager-cookbook into sclark/rubocop_fixes_add_badge

Signed-off-by: Steve Clark <steve@bigsteve.us>

sclark007 committed:
update merge and add dco

sclark007 committed:
# This is a combination of 2 commits.
# This is the 1st commit message:

fix travis

Signed-off-by: Steve Clark <steve@bigsteve.us>

# This is the commit message #2:

update readme

sclark007 committed:
fix travis

Signed-off-by: Steve Clark <steve@bigsteve.us>

sclark007 committed:
updates to for travis and gemfile

Signed-off-by: Steve Clark <steve@bigsteve.us>

sclark007 commented:
@jjasghar pretty sure i made things worse. this was my first attempt at using git to squash a commit. i can close the pull request and start over if needed. 

thanks,

jjasghar commented:
Closing due to the confusion. No worries though thanks for the PR, if you want to try again I'm more than willing to review another time.
